install.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
```

Comment:
It looks like it's your test script, which should not have been committed to the repo.